app/views/hyrax/dashboard/collections/_show_actions.html.erb

```diff
@@ -9,6 +9,14 @@
                   hyrax.my_works_path(add_files_to_collection: presenter.id),
                   title: t('hyrax.collection.actions.add_works.desc'),
                   class: 'btn btn-default' %>
+      <%# TODO - The presenter's solr document does not have any collection_type information.
```

Comment:
NEW: Does this comment still apply?  You added presenter.collection_type method.